protocol/statetracker/state_tracker.go

```diff
@@ -21,17 +21,23 @@ const (
 // ConsumerStateTracker CSTis a class for tracking consumer data from the lava blockchain, such as epoch changes.
 // it allows also to query specific data form the blockchain and acts as a single place to send transactions
 type StateTracker struct {
-	chainTracker         *chaintracker.ChainTracker
-	registrationLock     sync.RWMutex
-	newLavaBlockUpdaters map[string]Updater
-	EventTracker         *EventTracker
+	chainTracker          *chaintracker.ChainTracker
+	registrationLock      sync.RWMutex
+	newLavaBlockUpdaters  map[string]Updater
+	emergencyModeUpdaters map[string]EmergencyModeUpdater
```

Comment:
all updaters are in the same map, and need to implement Updater interface, your usage is wrong, convert EmergencyModeUpdater to implement Updater interface and use the regular function